src/form/acp.jl

```diff
@@ -56,6 +56,12 @@ function variable_mc_bus_voltage_on_off(pm::AbstractUnbalancedACPModel; nw::Int=
 end
 
 
+"Capacitor switching variables"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature